packages/coding-standard/src/Rules/PreferredSmartFileSystemOverNativeFileSystemFunctionCallRule.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\CodingStandard\Rules;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\FuncCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use Symplify\CodingStandard\PhpParser\NodeNameResolver;
+
+/**
+ * @see \Symplify\CodingStandard\Tests\Rules\PreferredSmartFileSystemOverNativeFileSystemFunctionCallRule\PreferredSmartFileSystemOverNativeFileSystemFunctionCallRuleTest
+ */
+final class PreferredSmartFileSystemOverNativeFileSystemFunctionCallRule extends AbstractCheckFunctionCallRule
```

Comment:
The rule name should be probably `PrefferedMethodCallOverFuncCallRule`

Reply:
implemented.